app/assets/csv/sample_reporting_payout.csv

```diff
@@ -0,0 +1,2 @@
+Reference,Description,Email,Amount,Card Brand,Cardholder Name,Card Expiry Date,Card Number,State,Finished,Error Code,Error Message,Provider ID,GOV.UK Payment ID,Issued By,Date Created,Time Created,Corporate Card Surcharge,Total Amount,Wallet Type,Fee,Net,Card Type,MOTO
+4S8CKS3FN2,test kb,useremail@example.org,10,Visa,test,11/21,4242,Success,true,,,pi_1HG0y2Hj08j2jFuBmM9r9Ybe,9mim88o2gt7atngu9m7i78en6a,,14 Aug 2020,11:20:48,0,10,,0.5,9.5,credit,false
```

Comment:
Could we put a more realistic test payment description, something like "Pay your council tax", and realist cardholder name etc. maybe lift from the API documentation? https://docs.payments.service.gov.uk/making_payments/#take-a-payment

Also, the amount should match the total on the payouts page. Have we talked about whether we want a number of transactions that add up to that amount to make this a more realistic report? The total amount on the payouts page should match the total net amount in the CSV